Source/WebKitLegacy/mac/History/BackForwardList.h

```diff
@@ -31,12 +31,13 @@
 #include <wtf/CheckedRef.h>
 #include <wtf/HashSet.h>
 #include <wtf/Vector.h>
+#include <wtf/WeakPtr.h>
```

Comment:
Can we remove the `Checked` header include?

Reply:
Yes, will drop.